api/v1alpha2/cloudprovider/types.go

```diff
@@ -65,6 +65,8 @@ type StorageConfig struct {
 	AttacherImage       string `json:"attacherImage,omitempty"`
 	ProvisionerImage    string `json:"provisionerImage,omitempty"`
 	MetadataSyncerImage string `json:"metadataSyncerImage,omitempty"`
+	LivenessProbeImage  string `json:"livenessProbeImage,omitempty"`
```

Comment:
Hi @jiatongw,

This line and the next should have `// +optional` as Kubernetes API guidelines state wherever there is `omitempty`,  the OpenAPI requires `// +optional` for said field.

Yes, I realize other fields are missing it, but that's because I need to go whip @andrewsykim with a wet piece of string and get him to update those fields :)

Reply:
Also, please add GoDoc comments for exported fields. Regarding the other fields, see the note about A Kim and a piece of wet string :)